.github/workflows/xrt_master_2021.1.yml

```diff
@@ -30,7 +30,13 @@ jobs:
             os_ver: ubuntu_20.04      
           - os: ubuntu2204      
             packageType: deb
-            os_ver: ubuntu_22.04      
+            os_ver: ubuntu_22.04 
+          - os: amazonlinux-20202011111 
+            os_ver: amazonlinux20202011111     
```

Comment:
Does the order matters? I see for AL2, packageType is after os_ver.

Reply:
It should not matter but I have changed it! Thanks